src/main/java/doob/game/model/Game.java

```diff
@@ -233,7 +153,13 @@ public void readLevels(String levelPath) {
 			NodeList levels = doc.getElementsByTagName("LevelName");
 			for (int i = 0; i < levels.getLength(); i++) {
 				Node n = levels.item(i);
-				String s = "src/main/resources/level/" + n.getTextContent();
+				String custom;
+				if (isCustom) {
+					custom = "/Custom";
+				} else {
```

Comment:
Shouldn't this be "Custom/"? Because if you look to string s (below) you will have a double slash on the one side and no slash on the other side